<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GamePageClass</class>
 <widget class="QWidget" name="GamePageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GamePage</string>
  </property>
  <widget class="QFrame" name="frameChat">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>251</width>
     <height>261</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QScrollArea" name="chatScrollArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>189</width>
       <height>189</height>
      </rect>
     </property>
     <widget class="QListWidget" name="answerList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>191</width>
        <height>191</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="chatInput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendAnswerBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
  </widget>
  <widget class="DrawingAreaWidget" name="drawingArea" native="true">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>651</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelWord">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>word</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRound">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fourth Round</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="undoLastLineBtn">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>560</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>undo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitGameBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetDrawingBtn">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>560</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="lableSeconds">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>60 s</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="leaderBoard">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>199</width>
     <height>199</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>PlayerName</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Score</string>
    </property>
   </column>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawingAreaWidget</class>
   <extends>QWidget</extends>
   <header location="global">DrawingAreaWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
